unit_test/blas/Test_Blas2_gemv.hpp

```diff
@@ -298,3 +440,19 @@ TEST_F(TestCategory, gemv_double_int) {
   // Kokkos::Profiling::popRegion();
 }
 #endif
+
+#if defined(KOKKOSKERNELS_INST_DOUBLE) || \
+    (!defined(KOKKOSKERNELS_ETI_ONLY) &&  \
+     !defined(KOKKOSKERNELS_IMPL_CHECK_ETI_CALLS))
+TEST_F(TestCategory, gemv_unified_interfaces) {
+  Kokkos::Profiling::pushRegion("KokkosBlas::Test::gemv_unified_interfaces");
+#if defined(KOKKOSKERNELS_INST_LAYOUT_LEFT)
+  ::Test::impl_test_gemv_unified_interface<TestExecSpace, Kokkos::LayoutLeft>();
```

Comment:
1. typos: `KOKKOSKERNELS_INST_LAYOUTLEFT` & `KOKKOSKERNELS_INST_LAYOUTRIGHT`
2. `impl_test_gemv_unified_interface()` doesn't take layout anyway (only the device)